src/common/components/Callout/style.css.ts

```diff
@@ -0,0 +1,19 @@
+import { style, styleVariants } from '@vanilla-extract/css';
+
+import { theme } from 'styles/theme.css';
+
+const containerBase = style({
+  display: 'flex',
+  gap: '22px',
+  alignItems: 'center',
+  width: 'fit-content',
+  padding: '28px 28px',
+  borderRadius: '15px',
+  backgroundColor: theme.color.subBackground,
+  whiteSpace: 'pre-line',
+});
```

Comment:
font 설정이 누락된 것 같아요.
```suggestion
const containerBase = style({
  display: 'flex',
  gap: '22px',
  alignItems: 'center',
  width: 'fit-content',
  padding: '28px 28px',
  borderRadius: '15px',
  backgroundColor: theme.color.subBackground,
  whiteSpace: 'pre-line',
    ...theme.font.TITLE_5_18_SB,
});
```
요친구 추가해주시면 될 것 같아용

Reply:
callout 안에서 사용되는 font의 종류가 다르기에 이를 여기서 설정하기 보다 사용되는 곳에서 설정하는 것이 좋을 거 같습니다!

<img width="1019" alt="스크린샷 2024-06-08 오후 5 46 16" src="https://github.com/sopt-makers/sopt-recruiting-frontend/assets/121864459/2f898f35-ca88-4304-8a7c-92195a939861">
<img width="472" alt="스크린샷 2024-06-08 오후 5 46 32" src="https://github.com/sopt-makers/sopt-recruiting-frontend/assets/121864459/2c85ba37-553a-4d6c-a17e-d30ca10d0a43">

Reply:
아하... 보니까 버튼만 폰트가 mds 적용이 안되어있어서 과연 디자인 분들이 의도하신 부분인지 궁금하네요.! 제가 질문드렸으니 답변 달리면 다시 코멘트달겠습니다

그런데 버튼 폰트가 다르다고 하더라도 나와있는 디자인상 callout font에 다양한 배리가 있는 상황이 아니기 때문에 최소한 default font는 설정해줘야한다고 생각하는데 어떻게 생각하실까요?? 
모든 텍스트는 일단 고정적인 font니까 저렇게 적용시켜주고, 
어차피 다른 폰트의 버튼이 추가될 경우 children 내부에 직접 스타일링해줄테니까 그때 font 스타일 오버라이딩해주면 될 것 같은데 어떻게 생각하시나요?

Reply:
좋습니다!

Reply:
반영했습니다!

Reply:
@eonseok-jeon 
요거 통일된다고 합니다!!
![image](https://github.com/sopt-makers/sopt-recruiting-frontend/assets/81505421/1dc552de-1705-49eb-81bd-943d5f3c70c2)

Reply:
오예~~~~